### PR TITLE
Lvbirgelen initial bastion session script

### PR DESCRIPTION
**Pull request by lvbirgelen**

### CODE REVIEW

**Comment by lvbirgelen** [comment]
I have read the CLA Document and I hereby sign the CLA

File: security/security-design/bastion-session-script/bastion-session.sh [hunk 1]
```diff
@@ -0,0 +1,499 @@
+#!/usr/bin/env bash
```
Comment (AlexanderHodicke) [inline security/security-design/bastion-session-script/bastion-session.sh:1]: Please move this file to a subfolder called /files

@oheimburger code review please

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]
please move the file and @oheimburger code review

File: security/security-design/bastion-session-script/bastion-session.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (lvbirgelen) [reply security/security-design/bastion-session-script/bastion-session.sh:1]: For portability its considered better to use the original:
https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability

**Pull request merged**